configure-make.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Good idea adding the `-make` version. To be consistent as part of this change it might be worth updating `configure.sh/bat` to both be called `configure-ninja.sh/bat` (with a `git mv`)